tests/warehouses/spark_suite.py

```diff
@@ -7,3 +9,15 @@ class SparkSuite(SqlTestSuite):
     def setUp(self) -> None:
         self.target = TARGET_SPARK
         super().setUp()
+
+    def test_sql_columns_metadata(self):
```

Comment:
I think this could be a test applied to **all** dialects. However, then we need to integrate `ColumnMetadata` with the core.